components/automate-ui/src/app/modules/infra-proxy/clients/clients.component.scss

```diff
@@ -1,5 +1,20 @@
 @import "~styles/variables";
 
+.search-create-container {
+  display: flex;
```

Comment:
Author note: Flexing allows us to easily change the width of our element should the user change the size of their browser window, without disturbing the design and user experience